src/util/Random.h

```diff
@@ -26,16 +26,12 @@ template <typename Int>
 requires(std::is_integral_v<Int> && sizeof(Int) <= sizeof(uint64_t))
 class FastRandomIntGenerator {
  public:
-  FastRandomIntGenerator() {
+  explicit FastRandomIntGenerator(unsigned int seed = std::random_device{}()) {
     // Randomly initialize the shuffleTable
-    std::random_device seeder{};
-    // `std::random_device` only yields 32 bit values, so we need two of them
-    // for each entry of `_shuffleTable`
-    static_assert(sizeof(decltype(seeder())) == 4);
+    std::default_random_engine randomEngine{seed};
```

Comment:
Use a named engine that is standardiczed, e.g. `std::mt19937_64`. 
That way you also get (more) consistent behaviors for different standard library implementations.

Reply:
Done, see [71ce001](https://github.com/ad-freiburg/qlever/pull/1126/commits/71ce00129b48bfd1512908208e47186986ffb777).

I hope it's alright, that I also replaced the other uses of `default_random_engine`.